### PR TITLE
Specify language attribute for DTS website's HTML

### PR DESCRIPTION
**Pull request by RR216**
This PR closes the first half of: https://github.com/cityofaustin/atd-data-tech/issues/12898
Here is the deploy preview: https://deploy-preview-67--atd-product.netlify.app/
Here are the details:
Add lang="en" to html element of our websites so they meet [W3C Web Content Accessibility Guidelines (WCAG) accessibility standards](https://webaim.org/standards/wcag/checklist#sc3.1.1).

### CODE REVIEW

**Review by amenity — APPROVED** [review]
🇬🇧 ✔️ 🚢

File: next.config.js
Comment (frankhereford) [inline next.config.js:8]: Before you merge this, would you mind adding a blank line at the end of this file? Thanks!

**Review by frankhereford — APPROVED** [review]
Works great - thank you Rachel!

image

**Review by mddilley — APPROVED** [review]
Great stuff! 🚢 Approving, but see Frank's suggestion above 🙏 (i didn't want to bury it with my approval)

**Pull request merged**
